### ISSUE TITLE
Error: Unknown command develop in gridsome

### ISSUE THREAD

**Issue by sriramgroot** [gridsome/gridsome#1250]
## Description

After installation of gridsome, when try to start running `gridsome develop`

### Steps to reproduce

1) Proceed with installation of Gridsome CLI tool
2) Next create gridsome project
3) Finally run command `gridsome develop`

### Expected result

`gridsome develop` to start local dev server at `http://localhost:8080`

### Actual result

This should be running the gridsome application in our localhost, but it showing below mentioned error.

### Environment

Paste the information here as shown by `gridsome info`
```
Srirams-MBP:mine sriramkarthick$ gridsome info

  System:
    OS: macOS Mojave 10.14.6
    CPU: (4) x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
  Binaries:
    Node: 12.13.1 - ~/.nvm/versions/node/v12.13.1/bin/node
    Yarn: 1.16.0 - /usr/local/bin/yarn
    npm: 6.12.1 - ~/.nvm/versions/node/v12.13.1/bin/npm
  Browsers:
    Chrome: 83.0.4103.116
    Edge: 83.0.478.56
    Firefox: 77.0.1
    Safari: 13.1.1
```

### Screenshot for your reference:
<img width="722" alt="Screenshot 2020-06-28 at 9 54 48 PM" src="https://user-images.githubusercontent.com/58930932/85952958-7a7f2600-b98a-11ea-8dd5-8010508c4234.png">

**Comment by thetre97** [comment]
@sriramgroot Does running `yarn develop` work?

**Comment by sriramgroot** [comment]
@thetre97 will check and get back to as soon as possible

**Comment by sriramgroot** [comment]
@thetre97 For `yarn develop` iam getting the same error, attached screenshot for your reference.
<img width="552" alt="Screenshot 2020-07-01 at 4 48 39 PM" src="https://user-images.githubusercontent.com/58930932/86238221-c31d2600-bbba-11ea-969c-b2dfbab836a6.png">

**Comment by thetre97** [comment]
@sriramgroot And what version of Gridsome & Gridsome CLI do you have installed?

**Comment by sriramgroot** [comment]
@thetre97 Iam currently using below mentioned version for **gridsome**

`@gridsome/cli v0.3.3` and `"gridsome": "^0.7.0"`

**Comment by Sergix** [comment]
Are you sure you're in the correct directory?

**Comment by thetre97** [comment]
Could you also try updating gridsome to the latest version, `0.17.8`?

**Comment by sriramgroot** [comment]
> Are you sure you're in the correct directory?

Hi @Sergix  Yes, Iam in the right directory

**Comment by sriramgroot** [comment]
> Could you also try updating gridsome to the latest version, `0.17.8`?

@thetre97 Sure will try that and get back to you as soon as possible

**Comment by a-kriya** [comment]
Seems like an issue with installation. Instead of using the globally installed CLI, can you try `npx gridsome develop` in the project root?

**Comment by datrinh** [comment]
Having the same issue. Wanted to give this a try, but kinda disheartening if this causes trouble so early on.

**Comment by sugardayfox** [comment]
Upgrading both node to LTS and gridsome to v0.7.8 made the trick for me! Thx @thetre97 !

**Comment by programmende** [comment]
Do a "npm install" before and it should work.

**Comment by YingjieQiao** [comment]
I'm having the same issue. However, when I run `gridsome --version` I only get 

```
@gridsome/cli v0.3.4
```

even thought I have run `npm --global install gridsome` and `npm install --global @gridsome/cli`

**Comment by Sanjeevcn** [comment]
> Do a "**npm install**" before and it should work.

worked for me.

**Comment by Luckey-Elijah** [comment]
I'm also having the same problem with the command
```shell
gridsome develop
```
Output:
```console
Unknown command develop


Usage: gridsome <command> [options]

Options:
  -v, --version            output the version number
  -h, --help               output usage information

Commands:
  create <name> [starter]  create a new website
  info                     output information about the local environment

  Run gridsome <command> --help for detailed usage of given command.
```

I also ran `npm install` in the correct directory, this didn't help either. 😞 

Version:
```console
gridsome --version
@gridsome/cli v0.3.4
```

**Comment by daverogers** [comment]
Yeah I just installed the latest gridsome version and had the same issues. Ripped Yarn out of the project and ran `npm install` and now `gridsome develop` works 🤷‍♂️ Yarn didn't want to globally install `yarn global add @gridsome/cli` either so there might just be something goofy going on with Yarn overall.

**Comment by david-gagne** [comment]
@sriramgroot I'm having this issue as well. I was able to overcome it by following a solution to another problem, but now it's happening again. Have you been able to find a solution to this problem?

**Comment by lagnaw** [comment]
I did a "npm install" before and it worked. What this is solving exactly ?

**Comment by rGradov** [comment]
try to install Vue js to solve this issue (or re install) use  `npm install -g @vue/cli` or npm update -g @vue/cli and then  `npm install --global @gridsome/cli`

**Comment by metacortexplus** [comment]
Same error. Tried `yarn run develop`, build starts but get compilation error eventually. I did install cli with pnpm. Worth to mention that instruction for pnpm here on github is wrong. It should be `pnpm add -g @gridsome/cli` (accordingly to docs pnpm doesn't have `-g` option for `install` at all). Also docs on Gridsome website doesn't contains instruction for pnpm at all.

**Comment by rasensio** [comment]
If this is a new project you just downloaded from git, run `npm i` or `npm install` to get started before running the `gridsome develop` cmd.
